pgmq-extension/sql/pgmq--1.3.3--1.3.4.sql

```diff
@@ -0,0 +1,66 @@
+-- Add function to convert archive queues to partitioned tables
+CREATE FUNCTION pgmq.convert_archive_partitioned(table_name TEXT,
+                                                 partition_interval TEXT DEFAULT '10000',
+                                                 retention_interval TEXT DEFAULT '100000',
+                                                 leading_partition INT DEFAULT 10)
+RETURNS void AS $$
+DECLARE
+a_table_name TEXT;
+a_table_name_old TEXT;
+qualified_a_table_name TEXT;
+qualified_a_table_name_old TEXT;
+BEGIN
+  PERFORM pgmq._ensure_pg_partman_installed();
+  SELECT FORMAT('%s', 'a_'|| table_name) INTO a_table_name;
+  SELECT FORMAT('%s', 'a_'|| table_name || '_old') INTO a_table_name_old;
+  SELECT FORMAT('%s.%s', 'pgmq','a_'|| table_name) INTO qualified_a_table_name;
+  SELECT FORMAT('%s.%s', 'pgmq','a_'|| table_name || '_old') INTO qualified_a_table_name_old;
```

Comment:
The trick is to use `%s` for strings and `%I` for identifiers. If you're just concatenating you don't need to use format. So:

```suggestion
a_table_name TEXT := 'a_' || table_name;
a_table_name_old TEXT := 'a_'|| table_name || '_old';
qualified_a_table_name TEXT := format('%I.%I', 'pgmq', table_name);
qualified_a_table_name_old TEXT := format ('%I.%I', 'pgmq', table_name || '_old');
BEGIN
```

Reply:
I see now that it's correct in pgmq.sql; just need to dupe it here.